services/eventservice/src/tests.rs

```diff
@@ -0,0 +1,508 @@
+            name: "N/A".to_string(),
+        },
+    };
+    let event3 = types::Event {
+        id: 3,
+        title: "Event3".to_string(),
+        cover: "N/A".to_string(),
+        description: "Empty".to_string(),
+        organizer: 2,
+        date_created: chrono::Utc::now().timestamp() + 10,
+        date_planning: (chrono::Utc::now() + chrono::Duration::days(1)).timestamp(),
+        category: Category {
+            name: "N/A".to_string(),
+        },
+    };
+    assert_eq!(
+        data_manager::EventManager::_events_to_vec(&vec![
+            event1.to_json().unwrap().as_bytes(),
+            event3.to_json().unwrap().as_bytes()
+        ]),
+        vec![event1, event3]
+    );
+}
+
+#[test]
+fn ids_to_vec_test() {
+    assert_eq!(
+        data_manager::EventManager::_ids_to_vec(&(42 as u64).to_be_bytes()),
+        vec![42 as u64]
+    );
+    assert_eq!(
+        data_manager::EventManager::_ids_to_vec(b"42"),
+        Vec::<u64>::new()
+    );
+}
+
+#[test]
+fn create_event_test() {
+    let manager = data_manager::EventManager::new(&"./test/createeventtestdb".to_string());
+    let event1 = types::Event {
+        id: 1,
+        title: "Event1".to_string(),
+        cover: "N/A".to_string(),
+        description: "Empty".to_string(),
+        organizer: 3,
+        date_created: chrono::Utc::now().timestamp(),
+        date_planning: (chrono::Utc::now() + chrono::Duration::days(2)).timestamp(),
+        category: Category {
+            name: "N/A".to_string(),
+        },
+    };
+    assert!(manager.create_event(&event1).is_err());
+    manager
+        .create_category(&Category {
+            name: "N/A".to_string(),
+        })
+        .expect("Failed to create category!");
+    manager
+        .create_event(&event1)
+        .expect("Failed to create event");
+    let event = manager.get_event(&1).expect("Failed to get event");
+    assert_eq!(event, event1);
+    manager._reset_all().expect("Failed to delete db");
+}
+
+#[test]
+fn delete_event_test() {
+    let manager = data_manager::EventManager::new(&"./test/deleventtestdb".to_string());
+    let event1 = types::Event {
+        id: 1,
+        title: "Event1".to_string(),
+        cover: "N/A".to_string(),
+        description: "Empty".to_string(),
+        organizer: 3,
+        date_created: chrono::Utc::now().timestamp(),
+        date_planning: (chrono::Utc::now() + chrono::Duration::days(2)).timestamp(),
+        category: Category {
+            name: "N/A".to_string(),
+        },
+    };
+    let event2 = types::Event {
+        id: 2,
+        title: "Event2".to_string(),
+        cover: "N/A".to_string(),
+        description: "Empty".to_string(),
+        organizer: 1,
+        date_created: chrono::Utc::now().timestamp(),
+        date_planning: (chrono::Utc::now() + chrono::Duration::days(2)).timestamp(),
+        category: Category {
+            name: "ABC".to_string(),
+        },
+    };
+    let event3 = types::Event {
```

Comment:
the same as before. Will you create events each time you add new test? It should be more dynamic